CHANGELOG.md

```diff
@@ -4,6 +4,83 @@ IMPROVEMENTS:
 * Use `consul-k8s` subcommand to perform `tls-init` job. This allows for server certificates to get rotated on subsequent runs.
   Consul servers have to be restarted in order for them to update their server certificates [[GH-749](https://github.com/hashicorp/consul-helm/pull/721)]
 
+BREAKING CHANGES:
```

Comment:
These are getting so long maybe I should just have:

* `connectInject.centralConfig.defaultProtocol` is no longer supported. See [Upgrade to CRDs](https://www.consul.io/docs/k8s/crds/upgrade-to-crds) for full instructions on how to upgrade.
* ...

Reply:
Yeah. I like the idea of keeping the CHANGELOG a little more succinct.